src/vs/editor/browser/viewParts/minimap/minimap.ts

```diff
@@ -453,6 +455,8 @@ export class Minimap extends ViewPart {
 	private _options: MinimapOptions;
 	private _lastRenderData: RenderData | null;
 	private _lastDecorations: ViewModelDecoration[] | undefined;
+	private _selections: Selection[] = [];
+	private _selectionColor: Color | undefined;
```

Comment:
small nitpick: please use `null` instead of `undefined` (to keep in spirit the existing code style)

Reply:
Ah, in this case the `undefined` is from the return type of `getColor`, so will leave as is